samza-test/src/test/java/org/apache/samza/test/samzasql/TestSamzaSqlEndToEnd.java

```diff
@@ -417,6 +416,8 @@ public void testEndToEndNestedRecord() {
     Assert.assertEquals(numMessages, outMessages.size());
   }
 
+  // SAMZA-2252 - Samza SQL subquery with flatten doesn't work after calcite 1.19
```

Comment:
This is required for Ambry use-case. What does it take to get it working ? Shall we wait on this change until the fix ?

Reply:
I fixed this issue. https://issues.apache.org/jira/browse/SAMZA-2252

Reply:
That's super quick fix. Thanks!